gloo/test/openssl_utils.cc

```diff
@@ -0,0 +1,94 @@
+#include "gloo/test/openssl_utils.h"
+
+#include <gloo/common/logging.h>
+
+#include <openssl/pem.h>
+#include <openssl/x509.h>
+
+#include <memory>
+
+namespace gloo {
+namespace test {
+
+#define wrap_in_unique_ptr_prefix(NAME, TYPE, PREFIX)                         \
```

Comment:
Curious, why macro name not in all capital letters? Is this the convention in Gloo code base?

Reply:
uppercased